cfgov/unprocessed/js/routes/sheer.js

```diff
@@ -0,0 +1,24 @@
+/* ==========================================================================
+   Common scripts that are used on old sheer templates
+   These should be removed as templates are moved to wagtail
+   ========================================================================== */
+
+'use strict';
+
+// List of modules often used.
+var FilterableListControls = require( '../organisms/FilterableListControls' );
+var Expandable = require( '../molecules/Expandable' );
+
+var filterableListDom = document.querySelector( '.o-filterable-list-controls' );
+var filterableListControls;
+for ( var i = 0, len = filterableListDom.length; i < len; i++ ) {
+  filterableListControls = new FilterableListControls( document.body )
+  filterableListControls.init();
+}
+
+var expandableDom = document.querySelectorAll( '.m-expandable' );
```

Comment:
There shouldn't be any pages. There are pages that use the `atomic expandable` but not the `atomic-expandable-group`.